contrib/babelfishpg_common/src/babelfishpg_common.h

```diff
@@ -27,6 +27,8 @@
 #define BBF_Pragma_IgnoreFloatConversionWarning_Pop \
     _Pragma("GCC diagnostic pop")
 
+enum TdsAtAtVarType {rcount_type, err_type, trancount_type};
```

Comment:
Please use the following format (following PG style)
```
typedef enum TdsAtAtVarType
{
RCOUNT_TYPE,
ERR_TYPE,
TRANCOUNT_TYPE
} TdsAtAtVarType;
```

Reply:
Good suggestion. Done.

Reply:
All comments are addressed.